doc/manual/tuto.md

```diff
@@ -0,0 +1,187 @@
+---
+slug: tutorial
+---
+
+# Managing users
+
+In this tutorial, you will learn how to use Nickel to manage a list of
+users, and then export it as a YAML file for a program consuming it.
```

Comment:
The "for a program consuming it" sounds a little strange to the ear, although I don't think it's incorrect per se. Maybe "for another program to consume"? Or just get rid of the end of the sentence?

```suggestion
users, and then export it as a YAML file.
```

Reply:
I wanted to give some kind of background about the "why we need to generate a YAML file", but I suppose we can drop the end of the sentence